modules/nf-core/cytotable/tests/main.nf.test

```diff
@@ -0,0 +1,40 @@
+nextflow_process {
+
+    name "Test Process CYTOTABLE"
+    script "modules/nf-core/cytotable/main.nf"
+    process "CYTOTABLE"
+
+    test("Should run without failures") {
+
+        when {
+            params {
+                // define parameters here.
+            }
+            process {
+                """
+                // define inputs of the process here. Example:
+                input[0] = [
+                    [batch: 'test_batch', plate: 'P1', well: 'A01'], // meta map
+                    file(params.cellprofiler_output + checkIfExists: true)
```

Comment:
Instead of using params we want to use direct urls/paths to the files we are using for testing.